Sources/SwiftFormat/API/Configuration+Dump.swift

```diff
@@ -0,0 +1,39 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2025 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
+
+extension Configuration {
+  /// Return the configuration as a JSON string.
+  public func asJsonString() throws -> String {
+    let data: Data
+
+    do {
+      let encoder = JSONEncoder()
+      encoder.outputFormatting = [.prettyPrinted]
+      if #available(macOS 10.13, *) {
```

Comment:
We have a deployment target of macOS 12, so `.sortedKeys` should always be available.